lib/logstash/inputs/dlq.rb

```diff
@@ -0,0 +1,41 @@
+require 'logstash/namespace'
+require 'logstash/inputs/base'
+require 'logstash-input-dlq.jar'
+
+
+class LogStash::Inputs::Dlq < LogStash::Inputs::Base
+  config_name 'dlq'
+
+  default :codec, 'plain'
+
+  config :path, :validate => :path, :required => true
+  config :sincedb, :validate => :string, :required => false
```

Comment:
By default this should be in `path.data/plugins/inputs/dead_letter_queue/`. That's what file input uses. Also rename to sincedb_path to match file input.